apps/flipper_auth/pubspec.yaml

```
@@ -40,6 +41,7 @@ dependency_overrides:
   archive: ^3.6.1  # unify archive across packages
   mime: ^2.0.0  # satisfy supabase_flutter's mime requirement
   sqlite3: ^3.1.0  # keep sqlite3 consistent
+  sqlite3: ^3.1.0  # keep sqlite3 consistent
```

YAMLlint (1.37.1)

[error] duplication of key "sqlite3" in mapping

(key-duplicates)